content/docs/user-guide/assets/pipeline/asset-dependencies-and-identifiers.md

```diff
@@ -7,16 +7,22 @@ toc: true
 ---
 
 Both source and product assets might have dependencies. Source asset and job dependencies are inputs to the **Asset Pipeline** and asset processing jobs. Product dependencies are outputs from the Asset Pipeline and are used to load and package product assets. Asset identifiers help ensure both types of asset dependencies are met.
+
+## Source dependencies
+
+Source dependencies are a process time concept.
+
+A source dependency triggers **Asset Processor** to rerun the process job whenever the file declared as a source dependency changes. For example, a `.assetinfo` file is a source dependency and triggers a process job on its source asset when it is modified.
```

Comment:
.assetinfo might actually be a bad example here because they're actually metafiles which work basically the same as source dependencies but are configured via data and don't use the same system...

Reply:
I'll switch it to another source dependency. I noticed that image files are declaring source dependencies on their assetinfo, but FBX files are not. We might want to cut a ticket on that later.